lib/server-code/events.js

```diff
@@ -225,6 +225,8 @@ addEvent(613, 'afterListing', FILE, ['path', 'pattern', 'recursive', 'pageSize',
 addEvent(614, 'beforeExists', FILE, ['path'])
 addEvent(615, 'afterExists', FILE, ['path', 'result'])
 
+addEvent(618, 'beforeDownload', FILE, ['fileURL'])
```

Comment:
is this a URL or relativePath?

Reply:
This is relativePath. Rename fileURL to relativePath?

Reply:
If you want rename it, so we must crete one more context block in console. Because now, context block has this logic:
```
requestFilePath: {
    ...
    code           : 'req.fileURL',
    ...
  },
```